src/index.ts

```diff
@@ -172,85 +178,44 @@ export class UltravoxSession {
     this.sendData({ type: 'input_text_message', text });
   }
 
-  mute(roles: Role[] | Set<Role>): void {
-    const roleSet = new Set<Role>(roles);
-
-    for (const role of roleSet) {
-      if (!Object.values(Role).includes(role)) {
-        throw new Error(`Invalid role: ${role}`);
-      }
-
-      switch (role) {
-        case Role.AGENT:
-          if (!this.room?.remoteParticipants) {
-            throw new Error('Cannot set isAgentMuted.');
-          }
-          this.isAgentMuted = true;
-          this.room.remoteParticipants.forEach((rp) => {
-            rp.audioTrackPublications.forEach((atp) => {
-              atp.track?.setMuted(this.isAgentMuted);
-            });
-          });
-          break;
-
-        case Role.USER:
-          if (!this.room?.localParticipant) {
-            throw new Error('Cannot set isCallerMuted.');
-          }
-          this.isUserMuted = true;
-          this.room.localParticipant.setMicrophoneEnabled(!this.isUserMuted);
-          break;
-
-        default:
-          break;
-      }
+  muteMic(): void {
+    if (!this.room?.localParticipant) {
+      throw new Error('Cannot muteMic.');
     }
+    this._isMicMuted = true;
+    this.room.localParticipant.setMicrophoneEnabled(!this._isMicMuted);
```

Comment:
```suggestion
    this.room.localParticipant.setMicrophoneEnabled(false);
```

(just seems more clear)